### ISSUE TITLE
not able to predict with loaded model using universal embedding

### ISSUE THREAD

**Issue by romass12** [Tony607/Keras-Text-Transfer-Learning#4]
Model architecture --->
adam=keras.optimizers.Adam(lr=0.0001)
early_stopping = keras.callbacks.ModelCheckpoint(filepath, monitor='val_acc', verbose=0, save_best_only=True, save_weights_only=False, mode='auto', period=10)
input_text = Input(shape=(1,), dtype="string")
embedding = Lambda(UniversalEmbedding, output_shape=(512, ))(input_text)
dense = Dense(1024)(embedding)
bnorm = BatchNormalization()(dense)
acti = Activation('relu')(bnorm)
pred = Dense(number_classes, activation='softmax')(acti)
model = Model(inputs=[input_text], outputs=pred)
model.compile(loss='categorical_crossentropy', optimizer=adam, metrics=['accuracy',f1score,sensitivity,precision])

I loaded entire model as  --->
    with open('universal_model.json', 'r') as f:
    model = model_from_json(f.read())
    model.load_weights('./universal_model.h5')

Now fro prediction -->
    predicts = model.predict(np.array(["hi"], dtype=object))
I get this error --->
    FailedPreconditionError (see above for traceback): Attempting to use uninitialized value 
    module/Encoder_en/DNN/ResidualHidden_3/projection
	     [[{{node module_apply_default/Encoder_en/DNN/ResidualHidden_3/projection/read}} = 
    Identity[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"] . 
   (module/Encoder_en/DNN/ResidualHidden_3/projection)]]

TF version -- 1.11.0
Keras -- 2.2.4
